src/avatar_action.cpp

```diff
@@ -860,6 +860,10 @@ void avatar_action::eat( avatar &you, const item_location &loc,
 void avatar_action::plthrow( avatar &you, item_location loc,
                              const cata::optional<tripoint> &blind_throw_from_pos )
 {
+    if( you.get_working_arm_count() <= 0 ) {
+        add_msg( m_info, _( "You need at least one arm to even consider throwing something." ) );
+        return;
+    }
```

Comment:
I think it's better to move this check to `can_wield` and check the result of it in `plthrow` and `wield` instead, to avoid code duplication.